docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/troubleshooting-tutorials/tutorial-troubleshooting-a-failed-deployment.md

```diff
@@ -7,39 +7,53 @@ template: troubleshooting-guide-template
 If a deployment fails and there are no errors in the deploy or build output, do the following:
 
 
-## 1. Check deploy logs
+## Deployment fails at the "Build_Push_if_not_exist" step 
 
-Check the build logs via Log groups as follows. Filter the log groups by `codebuild`. If the deployment fails at the `Deploy_Scheduler` step, after checking the `codebuild` logs, filter by `jenkins` and check the logs.
+###1. Check deploy logs
+
+Check the build logs via Log groups as follows. Filter the log groups by `codebuild`. 
 
 {% include searching-by-logs.md %} <!-- To edit, see /_includes/searching-by-logs.md -->
 
+###2. Check multiple log groups at once
 
+Check multiple log groups via Logs Insights as follows. Select the log groups containing `codebuild`.
 
-## 2. Check multiple log groups at once
+{% include searching-by-multiple-log-groups.md %} <!-- To edit, see /_includes/searching-by-multiple-log-groups.md -->
 
-Check multiple log groups via Logs Insights as follows. Select the log groups containing `codebuild`. If the deployment fails at the `Deploy_Scheduler` step, also select the groups containing `jenkins`.
+###3. If you couldn't find any error messages in the logs check "Phase details" tab
 
-{% include searching-by-multiple-log-groups.md %} <!-- To edit, see /_includes/searching-by-multiple-log-groups.md -->
+1.Click on execution **"Details"**
 
-## 3. Check the status of services and tasks
+![execution_details]
 
-If the deployment fails at the `Deploy_Spryker_services` step, check the failed services and tasks for errors as follows:
+2. Click on **"Link to execution details"**
 
-{% include checking-the-status-of-ecs-services-and-tasks.md %} <!-- To edit, see /_includes/checking-the-status-of-ecs-services-and-tasks.md -->
+![link_to_execution_details]
 
+3. Click on **"Phase details"** tab
 
+![phase_details]
```

Comment:
What information should I look for in this tab?

Reply:
Status column != Succeeded

Reply:
on the screenshot that I have sent you "Time out" error